use_case_guide/multi_tenant.rst

```diff
@@ -438,19 +438,11 @@ First log in to the `Citus Console <https://console.citusdata.com/>`_ and open t
 
 .. image:: ../images/cloud-formation-configuration.png
 
-To add nodes, click "Change node count and size." A slider will appear for both the count and size. In this section we'll be changing only the count. You can learn more about other options in the [Cloud Scaling] section.
-
-.. image:: ../images/cloud-nodes-slider.png
-
-Drag the slider to increase node count by one, and click "Resize Formation." While the node is added the Cloud Console will display a message at the top of the screen:
-
-.. image:: ../images/cloud-change-progress.png
-
-.. note::
+To add nodes, click "Change node count and size." A slider will appear for both the count and size. For this section we'll be changing only the count.
 
-  Don't forget that even when this process finishes there is more to do! The new node will be available in the system, but at this point no tenants are stored on it so **Citus will not yet run any queries there**. Below we'll explore how to move existing data to the new nodes.
+Follow the instructions in :ref:`Cloud Scaling <cloud_scaling>` to add one more node to your cluster.
```

Comment:
I made this its own paragraph to cast it into the spotlight, but we may want to do more.